addons/web/static/src/core/hotkeys/hotkey_service.js

```diff
@@ -164,8 +164,11 @@ export const hotkeyService = {
             for (const el of getVisibleElements(ui.activeElement, "[data-hotkey]:not(:disabled)")) {
                 const hotkey = el.dataset.hotkey;
                 const overlay = document.createElement("div");
-                overlay.className = "o_web_hotkey_overlay";
-                overlay.appendChild(document.createTextNode(hotkey.toUpperCase()));
+                const overlay_kbd = document.createElement("kbd");
+                overlay.className = "o_web_hotkey_overlay position-absolute d-flex justify-content-center align-items-center m-0 bg-black-50 h6";
```

Comment:
`h6` `m-0` are probably unnecessary now

Reply:
@stefanorigano 
After some testing, the `h6` is still needed to keep the font size fixed according to its parent. So we also need to keep the `m-0` to remove the margin of the `h6` in order to have a full height for each hotkey.

![demo_h6](https://user-images.githubusercontent.com/80679690/152498162-9f2553a1-f876-4745-b8e3-36f6e0d791c8.jpg)

Reply:
Yes, I now see that `$kbd-font-size` is not an arbitrary `rem` value, but an `em` instead.
Let's keep `h6 m-0` 👍

Reply:
Well actually, I think that it answer one of my question below :grin: